packages/world/test/utils/TestUtils.sol

```diff
@@ -55,23 +57,28 @@ library TestUtils {
     return address(bytes20(vm.load(address(this), LIB_ADDRESS_SLOT)));
   }
 
-  function addToInventoryCount(
-    EntityId ownerEntityId,
-    ObjectTypeId ownerObjectTypeId,
-    ObjectTypeId objectTypeId,
-    uint16 numObjectsToAdd
-  ) public asWorld {
-    _addToInventoryCount(ownerEntityId, ownerObjectTypeId, objectTypeId, numObjectsToAdd);
+  function addToInventory(EntityId ownerEntityId, ObjectTypeId objectTypeId, uint16 numObjectsToAdd) public asWorld {
+    require(!objectTypeId.isTool(), "To add a tool, you must use addToolToInventory");
+    _addToInventoryCount(ownerEntityId, ObjectType._get(ownerEntityId), objectTypeId, numObjectsToAdd);
   }
 
-  function removeFromInventoryCount(
+  function addToolToInventory(EntityId ownerEntityId, ObjectTypeId toolObjectTypeId) public asWorld returns (EntityId) {
+    return _addToolToInventory(ownerEntityId, toolObjectTypeId);
+  }
+
+  function removeFromInventory(
     EntityId ownerEntityId,
     ObjectTypeId objectTypeId,
     uint16 numObjectsToRemove
   ) public asWorld {
+    require(!objectTypeId.isTool(), "To remove a tool, you must pass in the tool entity id");
     _removeFromInventoryCount(ownerEntityId, objectTypeId, numObjectsToRemove);
```

Comment:
should we rename the original functions too?

Reply:
yee renamed!